flair/datasets/sequence_labeling.py

```diff
@@ -443,6 +444,161 @@ def __init__(
             **corpusargs,
         )
 
+    @staticmethod
+    def get_token_level_label_of_each_token(sentence: Sentence, label_type: str) -> List[str]:
+        """Generates a label for each token in the sentence. This function requires that the labels corresponding to the label_type are token-level tokens.
+
+        Args:
+            sentence: a flair sentence to generate labels for
+            label_type: a string representing the type of the labels, e.g., "pos"
+        """
+        list_of_labels = ["O" for _ in range(len(sentence.tokens))]
+        for label in sentence.get_labels(label_type):
+            label_token_index = label.data_point._internal_index
```

Comment:
Not so much a comment on this PR, but I wonder why this member is protected when it is needed sometimes outside of the class